scripts/ingests/filip_15.py

```diff
@@ -0,0 +1,133 @@
+# script to get the data from table 9 of Filippazzo 2015 paper for modeled parameters
+
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+
+from astropy.table import Table
+
+SAVE_DB = False  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+VERBOSE = False
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+# WRITE THE JSON FILES
+if SAVE_DB:
+    db.save_database(directory='data/')
+
+columns = ['Designation', 'Radius', 'log(g)', 'T_eff', 'Mass']  # columns wanted
+with open("scripts/ingests/table_9_filip_15.txt") as f:
+    content = f.readlines()
+    table_9 = Table.read(content, format='ascii', data_start=4, data_end=201, header_start=2, delimiter='\t',
+                         guess=False, fast_reader=False, include_names=columns)
+
+data_columns = ['Radius', 'log(g)', 'T_eff', 'Mass']  # columns with data values
+
+for i in table_9:  # rewriting so that the empty values are None
+    for c in data_columns:
+        if i[c] == 'cdots':
+            i[c] = None
```

Comment:
```suggestion
for row in table_9:  # rewriting so that the empty values are None
    for column in data_columns:
        if row[column] == 'cdots':
            row[column] = None
```